### PR TITLE
A new test for CASSANDRA-12582

### PR DESCRIPTION
**Pull request by stef1927**
A new test for reproducing the problem described in [CASSANDRA-12582](https://issues.apache.org/jira/browse/CASSANDRA-12582).

It passes locally for 2.1 and 2.2 without any code changes, but for 3.0 and higher it will fail until the ticket is committed.

### CODE REVIEW

File: schema_test.py [hunk 1]
```diff
@@ -140,6 +140,36 @@ def drop_column_and_restart_test(self):
         session.execute("USE ks")
         assert_one(session, "SELECT * FROM t", [0, 0])
 
+    def drop_static_column_and_restart_test(self):
+        """
+        Dropping a static column caused an sstable corrupt exception after restarting, here
+        we test that we can drop a static column and restart safely.
+
+        @jira_ticket CASSANDRA-11050
```
Comment (ptnapoleon) [inline schema_test.py]: This is the wrong ticket number?

Comment (stef1927) [inline schema_test.py]: Fixed, thank you.

**Comment by ptnapoleon** [comment]
+1

**Comment by stef1927** [comment]
[CASSANDRA-12582](https://issues.apache.org/jira/browse/CASSANDRA-12582) has been committed, merging.

**Pull request merged**